independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/UpdateProjectCommandHandler.java

```diff
@@ -59,7 +59,10 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                 invocation.getQuarkusProject().getExtensionsCatalog());
         final ArtifactCoords projectQuarkusPlatformBom = getProjectQuarkusPlatformBOM(currentState);
         if (projectQuarkusPlatformBom == null) {
-            invocation.log().error("The project does not import any Quarkus platform BOM");
+            invocation.log().error(
+                    "The project does not include the Quarkus platform BOM (io.quarkus.platform:quarkus-bom) in its dependencies. Common issues are:\n"
```

Comment:
That's not exactly true because `io.quarkus:quarkus-bom` is the BOM we recommend for extensions and it's valid too.
I'll try to think of a better wording but feel free to try :).

Reply:
It could also be any other platform. Ie. Com.redhat.quarkus.platform.